KonyAccelerate/modules/globalMod.js

```diff
@@ -0,0 +1,42 @@
+var thisForm=null;
+var debugLogs=false;
+var devWidth=null;
+
+function globalPreshow(){
+  thisForm=kony.application.getCurrentForm();
+  egLoggerPreshow();
+  egLogger("thisForm = "+thisForm.id);
+}
+
+function assignFormEvents(){
+  //thisForm.preShow=function(){frmCardListPreshow();};
```

Comment:
Remove commented code

Reply:
Will discuss with Ed and will remove after the discussion